src/App.js

```diff
@@ -1,13 +1,48 @@
 import React from "react";
-
+import Heading from "./Heading.js";
+import TouristInfoCards from "./TouristInfoCards.js";
 import Bookings from "./Bookings";
+import Footer from "./Footer.js";
 import "./App.css";
 
+const cities = [
```

Comment:
Hi @FarhanaFKhan, loved that you've abstracted the cities data into an array here.